index.rst

```diff
@@ -8,27 +8,24 @@ FLINT Handbook
 
 "There is nothing permanent except change. - Heraclitus ". 
 
-Change is a phenomenon we are taught to expect or welcome, but what should we do if those changes bring about adverse and unfortunate events? An excellent example of such a change is climate change. 
+We are trained to expect or welcome change, but what should we do if those changes result in unfavourable or unfavorable circumstances? Climate change is a prime illustration of such a change.
 
-Climate change can be referred to as long-term temperature and weather pattern shifts. Although these shifts may be natural, human activities like burning fossil fuels and deforestation have been the major cause of climate change since the 1800s. Other causes of climate change include solar variability, volcanic activity, etc.
+Long-term changes in temperature and weather patterns are referred to as climate change. Although these changes may be natural, since the 1800s, human actions like the combustion of fossil fuels and deforestation have been the primary contributors to climate change. Solar variation, volcanic activity, and other factors are some other natural sources of climate change.
 
-Land use associated with agriculture, forestry, and human development results in the emission of greenhouse gases that envelop the earth. Fossil fuels are a major source of emissions, but they are not the focus of FLINT.
```

Comment:
Combined the paragraphs so that the flow doesn't break.

Reply:
> This pull request is a few suggestions for the introduction section of the FLINT handbook.
> 
> The introduction written by Amarachi is SO GOOD. I did not feel lost as a reader.
> 
> I am adding a few comments via review to add a few more suggestions to make a reader feel more comfortable/to explain why the suggestion.

thank you so much!